widgets/FTPSource-batchsource.json

```diff
@@ -151,6 +185,42 @@
               "label": "False"
             }
           }
+        },
+        {
+          "widget-type": "toggle",
+          "label": "Terminate If Empty Row",
```

Comment:
this sounds a bit unclear and technical for a toggle label:

some suggestions:

- `Stop Reading at First Empty Row`
- `Stop on First Empty Row`
- `Terminate Reading After Empty Row`

Reply:
Took reference from this: https://github.com/cdapio/hydrator-plugins/blob/f6363b0d4bf7e753c11f946c3594549f0acc4104/core-plugins/widgets/Excel-batchsource.json#L100

Reply:
I think we should change at both places then to make it consistent and human readable everywhere.